view/frontend/web/js/view/payment/method-renderer/creditcard.js

```diff
@@ -155,13 +181,18 @@ define(
                     return true;
                 },
 
+                isCredicardGroupMode: function () {
+                    return window.checkoutConfig.payment.buckaroo.creditcard.groupCreditcards !== undefined && window.checkoutConfig.payment.buckaroo.creditcard.groupCreditcards == 1 ? true : false;
```

Comment:
why not do `return window.checkoutConfig.payment.buckaroo.creditcard.groupCreditcards === 1`